### PR TITLE
DPC-684 - As an internal user, I want to assign users to an organization.

### PR DESCRIPTION
**Pull request by switzersc-usds**
**Why**

Users should belong to an organization. A CMS user should be able to assign users to orgs. 

**What Changed**
Internal users can now assign users to organizations.

When no user is assign:
Screen Shot 2019-10-22 at 11 24 22 AM

Unassigned user page:
Screen Shot 2019-10-22 at 11 24 31 AM

Edit user page with org assignment field:
Screen Shot 2019-10-22 at 11 24 38 AM

User page with assigned org:
Screen Shot 2019-10-22 at 11 25 27 AM

Index of all users including assigned user:
Screen Shot 2019-10-22 at 11 25 34 AM

Assigned user page:
Screen Shot 2019-10-22 at 11 26 20 AM

**Choices Made**

Even though it looks like a user only has one organization in these screenshots, the backend architecture supports a many to many association so that a user can be associated with multiple orgs. In the future we can use the join model, OrganizationUserAssignments, to specify a user's role in that given organization, or any other information about the user's relationship with the org. 

**Tickets closed**:

DPC-684

**Future work**

- Mockups for multiple org assignment support
- implementing multiple org assignment features
- DPC-714 : filter by assigned org (didn't do that here yet)

### CODE REVIEW

**Comment by switzersc-usds** [comment]
There's a rubocop issue with a line too long but I'm going to leave it.

File: dpc-web/app/services/user_search.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    end
+
+    scope
+  end
+
+  def apply_keyword_search(scope)
+    if params[:keyword].present?
+      keyword = "%#{params[:keyword].downcase}%"
+      scope = scope.where(
+        'LOWER(users.first_name) LIKE :keyword OR LOWER(users.last_name) LIKE :keyword OR LOWER(users.email) LIKE :keyword',
```
[the remark this answers is not in the document]
Comment (switzersc-usds) [reply dpc-web/app/services/user_search.rb]: I know! But I ran into issues changing the string syntax and moved on. I can spend some time reworking this but seems less important than starting the token work.

File: dpc-web/db/schema.rb [hunk 2]
```diff
@@ -10,7 +10,15 @@
...
-ActiveRecord::Schema.define(version: 2019_09_18_195851) do
+<<<<<<< HEAD
+<<<<<<< HEAD
+ActiveRecord::Schema.define(version: 2019_10_17_201830) do
+=======
+ActiveRecord::Schema.define(version: 2019_10_07_162013) do
+>>>>>>> f79276e2... User belongs_to Organization
+=======
+ActiveRecord::Schema.define(version: 2019_10_07_162013) do
+>>>>>>> c4520fe7... User belongs_to Organization
```
Comment (switzersc-usds) [inline dpc-web/db/schema.rb]: Whoops, need to fix this.

**Comment by jaredcunha-usds** [comment]
oops, accidentally clicked the close button. sorry

**Comment by switzersc-usds** [comment]
We want to improve the workflow to allow org creation from the user page, but that will be a separate pull request.

**Pull request merged**